desktop/notification/fdo_test.go

```diff
@@ -28,111 +28,37 @@ import (
 	"github.com/godbus/dbus"
 	. "gopkg.in/check.v1"
 
-	"github.com/snapcore/snapd/dbusutil"
-	"github.com/snapcore/snapd/dbusutil/dbustest"
 	"github.com/snapcore/snapd/desktop/notification"
-	"github.com/snapcore/snapd/logger"
+	"github.com/snapcore/snapd/desktop/notification/notificationtest"
```

Comment:
Having read the other side, this is really nice! The tests are much easier to follow here.